src/msw/window.cpp

```diff
@@ -3382,6 +3426,17 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
             }
         }
         break;
+
+        case WM_TOUCH:
+        {
+            if ( !GestureFuncs::IsOk() )
+                break;
+
+            HandleTouch(wParam, lParam);
+            // Let DefWindowProc handle the touch events too
```

Comment:
Again, IMO we should only let unprocessed events through.

Reply:
DefWindowProc takes care of memory cleanup. 

In regard to platform agnostic behavior: GTK and OS X does not allow returning any handled flag for an event, so I wouldn't do it for a multi-platform solution.